core/domain/collection_services.py

```diff
@@ -1092,6 +1092,9 @@ def save_new_collection_from_yaml(committer_id, yaml_content, collection_id):
         committer_id: str. ID of the committer.
         yaml_content: str. The yaml content string specifying a collection.
         collection_id: str. ID of the saved collection.
+
+    Returns:
+        collection. The collection object.
```

Comment:
The type is `Collection` (the domain object).

Reply:
Done.